x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/expanded_row.tsx

```diff
@@ -42,39 +49,59 @@ type Item = SectionItem;
 interface Props {
   item: TransformListRow;
   onAlertEdit: (alertRule: TransformHealthAlertRule) => void;
+  transformsStatsLoading: boolean;
 }
 
-type StateValues = Optional<TransformListRow['stats'], 'stats' | 'checkpointing'>;
+const NoStatsFallbackTabContent = ({
+  transformsStatsLoading,
+}: {
+  transformsStatsLoading: boolean;
+}) => {
+  const { euiTheme } = useEuiTheme();
 
-export const ExpandedRow: FC<Props> = ({ item, onAlertEdit }) => {
-  const hideNodeInfo = useIsServerless();
+  const content = transformsStatsLoading ? (
+    <EuiLoadingSpinner />
+  ) : (
+    <EuiText textAlign="center" color="subdued" size="s">
```

Comment:
Did you consider using a warning `EuiCallout` here?

Reply:
Changed to using a warning callout here https://github.com/elastic/kibana/pull/166320/commits/77794843d7dee700c973f17e89eb2c7ae6bffa74

<img width="1149" alt="Screen Shot 2023-09-14 at 10 25 48" src="https://github.com/elastic/kibana/assets/43350163/a30afb25-23d3-4366-b6f1-2e49775a7c77">